src/Shaders/default.fragment.fx

```diff
@@ -476,10 +476,12 @@ color.rgb = max(color.rgb, 0.);
 
 #define CUSTOM_FRAGMENT_BEFORE_FRAGCOLOR
 #ifdef PREPASS
+	float writeGeometryInfo = color.a > 0.4 ? 1.0 : 0.0;

```

Comment:
Would it be advisable to have a #define constant instead of 0.4? I'm not sure, just asking.

Reply:
we could add a define `ALPHA_GEOMETRY_CUTOFF` in `Material.ts`

Reply:
it should also be accessible from `GeometryBufferRenderer`